docs/website/docs/pipelines/Hubspot.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Hubspot
+description: dlt pipeline for Hubspot API
+keywords: [hubspot api, hubspot pipeline, hubspot]
+---
+
+# HubSpot
+
+HubSpot is a customer relationship management (CRM) software and inbound marketing platform that helps businesses to attract visitors, engage customers, and close leads. 
+
+The `dlt` HubSpot pipeline allows you to automatically load data from HubSpot into a [destination](https://dlthub.com/docs/destinations) of your choice. It loads data from the following endpoints:
+
+|API|Data|
+| --- | --- |
+| Contacts | visitors, potential customers, leads |
+| Companies | information about organizations  |
+| Deals | deal records, deal tracking |
+| Products | goods, services |
+| Tickets | request for help from customers or users |
+| Quotes | pricing information of a product |
+| Web analytics  | events |
+
+## Get API credentials
+
+```
```

Comment:
```suggestion
:::info
```

Reply:
@AmanGuptAnalytics @rahuljo we should use [Admonitions](https://docusaurus.io/docs/markdown-features/admonitions) for things like this